hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/CsvDFSSource.java

```diff
@@ -125,4 +134,12 @@ public CsvDFSSource(TypedProperties props,
       return Option.empty();
     }
   }
+
+  @Override
```

Comment:
I understand atleast for kafka it may not be applicable. but since we repeat this for almost all sources, can we move this to Source class itself. All impls just need to update inputFiles list alone. rest will be taken care by Source.postCommit(). this also means, that you need to declare fileCleaner and inputList in Source class.

Reply:
Agreed this is not desirable. However it  is quite tricky since the current Source base classes are primarily based on the input format (Row vs String vs Avro) instead of the actual source (DFS vs Kafka, etc).  Same problem with the repeated pathSelector for each DFS Sources. Some refactoring may be required in order to abstract this cleanly

Reply:
got it, thanks. Can you please file a jira and assign it to me. (shivnarayan is my handle)